pkg/ui/changelog/changelog.go

```diff
@@ -0,0 +1,64 @@
+package changelog
+
+import (
+	"errors"
+
+	tea "github.com/charmbracelet/bubbletea"
+	changelog "github.com/futurice/jalapeno/pkg/ui/changelog/prompt"
+)
+
+type Changelog struct {
+	Increment string
+	Msg       string
+}
+
+func RunChangelog() (Changelog, error) {
+	verInc, err := runSelectPrompt()
+
+	if err != nil {
+		return Changelog{}, errors.New("failed to get version type")
+	}
+
+	logmsg, err := runTextAreaPrompt()
+
+	if err != nil {
+		return Changelog{}, errors.New("failed to get log message")
```

Comment:
Same as on previous error:
```suggestion
  return Changelog{}, fmt.Errorf("failed to get log message: %w", err)
```